src/cf-forms/src/atoms/legend.less

```diff
@@ -0,0 +1,10 @@
+.a-legend {
+    .h4();
+
+    // Legends do not wrap in IE.
+    // Different styles are required to ensure wrapping in different versions.
+    box-sizing: border-box; // IE9-11 & Edge 12-13
+    display: table; // IE8-11
```

Comment:
This is probably fine, but elsewhere we have things like this wrapped in .lt-ie classes. The one thing would be if we dropped support for a particular IE we could search for the classes associated with that version.